src/table/utils/styling-utils.js

```diff
@@ -65,6 +65,23 @@ export function getHeadStyle(layout, theme) {
 }
 
 export function getBodyStyle(layout, theme) {
+  const contentHoverBackgroundColor = theme.getStyle('object', 'straightTable.content.hover', 'backgroundColor');
+  const contentBackgroundColor = theme.getStyle('object', 'straightTable.content', 'backgroundColor');
+  const tableBackgroundColor = theme.getStyle('object', 'straightTable', 'backgroundColor');
+  const objectBackgroundColor = theme.getStyle('object', '', 'backgroundColor');
+  const isHoverBackgroundColorFromTheme = !(
+    contentHoverBackgroundColor === contentBackgroundColor ||
+    contentHoverBackgroundColor === tableBackgroundColor ||
+    contentHoverBackgroundColor === objectBackgroundColor
+  );
+
+  const contentHoverFontColor = theme.getStyle('object', 'straightTable.content.hover', 'color');
+  const contentFontColor = theme.getStyle('object', 'straightTable.content', 'color');
+  const tableFontColor = theme.getStyle('object', 'straightTable', 'color');
+  const isHoverFontColorFromTheme = !(
+    contentHoverFontColor === contentFontColor || contentHoverFontColor === tableFontColor
+  );
+
```

Comment:
To check that the contentHoverBackgroundColor and contentHoverFontColor are really from object->straightTable->content->hover->backgroundColor and color

Reply:
What do the properties on theme mean? How is it that if the content or table color is the same as the hover color, then it is a valid color. Can't you set one color for the content, then a different hover color?

Reply:
Wait there is a `!` in there. So if they are not the same, then it is a "real" hover color?